Assets/HoloToolkit/UX/Shaders/3DTextShader.shader

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+// Upgrade NOTE: upgraded instancing buffer 'Props' to new syntax.
```

Comment:
Do we need to keep these upgrade notes?

Reply:
I thought they might be useful, but i can removed if needed.

Reply:
Maybe add the comment to the PR description.
Every time Unity goes and changes their shader syntax, these notes get littered throughout the affected shaders. Over time, they lose their cachet.